### PR TITLE
Tests changed, move them to test-datasets

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ypriverol** [comment]
Please do not merge yet.

**Review by jpfeuffer — COMMENTED** [review]
Nice. Can you refactor the contrast parsing logic as well?

**Comment by ypriverol** [comment]
@jpfeuffer I have reversed the changes because testing `here` in conda it works but for containers we may need to create a multi-tools container in biocontainers using the Msstats + here, should we go in that route or leave two scripts with repeated code?

File: conf/test_full.config [hunk 1]
```diff
@@ -1,36 +1,36 @@
 /*
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Nextflow config file for running full-size tests
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Defines input files and everything required to run a full size pipeline test.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests (LFQ)
```
Comment (jpfeuffer) [inline conf/test_full.config:3]: Why is this file changed and why is called full if it does minimal tests?

Comment (ypriverol) [reply conf/test_full.config:3]: I think it is good to have the tmt and the full ci/cd tests. We where using the tmt as full. Then, I renamed the full test to tmt and as the full test I suggest to use now the LFQ (same code) but in the future we can put some other information. What do you think?

File: bin/msstats_plfq.R [hunk 2]
```diff
@@ -106,127 +196,33 @@ processed.quant <- dataProcess(quant, censoredInt = 'NA', featureSubset = args[6
...
-        }
-    } else {
-        contrast_lst <- unlist(strsplit(contrast_str,";"))
-        contrast_mat <- make_contrasts(contrast_lst, lvls)
-    }
-
+    contrast_mat <- parse_contrasts(l = l, contrast_str = contrast_str, lvls = lvls)
     print ("Contrasts to be tested:")
     print (contrast_mat)
     #TODO allow for user specified contrasts
```
Comment (jpfeuffer) [inline bin/msstats_plfq.R:206]:
```suggestion
```

File: modules/local/msstats/main.nf [hunk 3]
```diff
@@ -1,7 +1,7 @@
 process MSSTATS {
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::bioconductor-msstats=4.2.0" : null)
+    conda (params.enable_conda ? "bioconda::bioconductor-msstats=4.2.0 conda-forge::r-here" : null)
```
Comment (jpfeuffer) [inline modules/local/msstats/main.nf:4]:
```suggestion
    conda (params.enable_conda ? "bioconda::bioconductor-msstats=4.2.0" : null)
```

**Comment by jpfeuffer** [comment]
Last thing, can you refactor msstats_tmt in the same way? E.g. put the functions on the top with a comment?

**Comment by ypriverol** [comment]
> Last thing, can you refactor msstats_tmt in the same way? E.g. put the functions on the top with a comment?

Last commit!!!

**Comment by ypriverol** [comment]
@jpfeuffer take a look

**Review by jpfeuffer — APPROVED** [review]
